pkg/features/kube_features.go

```diff
@@ -163,14 +155,6 @@ const (
 	// Enables SecretRef field in CSI NodeExpandVolume request.
 	CSINodeExpandSecret featuregate.Feature = "CSINodeExpandSecret"
 
-	// owner: @pohly
-	// alpha: v1.19
-	// beta: v1.21
-	// GA: v1.24
-	//
-	// Enables tracking of available storage capacity that CSI drivers provide.
-	CSIStorageCapacity featuregate.Feature = "CSIStorageCapacity"
```

Comment:
I would put CSIStorageCapacity removal in a different PR and make this one only about gcepd.

Reply:
How about putting it in https://github.com/kubernetes/kubernetes/pull/114068 ?

Reply:
Looks like #114068 is already merged.
There's another PR submitted to remove CSIStorageCapacity feature gate: https://github.com/kubernetes/kubernetes/pull/118018
The title for this PR is "remove CSI-migration gate" so it's better to only remove CSI-migration related things.

Reply:
yes, this PR only removes CSI-migration related things.